services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -188,37 +190,48 @@ async def remove_disconnected_user_resources(
                     dead_key,
                 )
 
+                can_remove_all = True
                 if resource_name == "project_id":
                     # inform that the project can be closed on the backend side
                     #
                     try:
-                        await remove_project_interactive_services(
+                        await remove_project_dynamic_services(
                             user_id=int(dead_key["user_id"]),
                             project_uuid=resource_value,
                             app=app,
-                            user_name={
-                                "first_name": "garbage",
-                                "last_name": "collector",
-                            },
+                            notify_users=True,
                         )
 
-                    except ProjectNotFoundError as err:
+                    except (
+                        ProjectNotFoundError,
+                        UserNotFoundError,
+                    ) as err:
                         logger.warning(
-                            (
-                                "Could not remove project interactive services user_id=%s "
-                                "project_uuid=%s. Check the logs above for details [%s]"
-                            ),
-                            user_id,
+                            "Could not remove dynamic services for project with %s project_uuid=%s [%s]",
+                            f"{user_id=}",
                             resource_value,
                             err,
                         )
+                        can_remove_all = True
```

Comment:
yes, the default is true, then depending on the handler is becomes True or False. 

Actually the problem is that I forgot to remove the ``with suppress(ProjectLockError)`` in ``remove_project_dynamic_services``. I agreed with @sanderegg  that it should be raised so the caller can
identify the error instead of silently failing